src/js/constants.ts

```diff
@@ -95,7 +95,7 @@ export const TEXTS = {
   HELP_SECTION: 'the help section',
   INDEX_SECTION: 'the index page',
   GENERIC_ERROR: `ਵਾਹਿਗੁਰੂ! Something isn't working correctly`,
-  GENERIC_ERROR_DESCRIPTION: `This is really unusual. You may want to hit the Feedback button at the bottom of this page, and report this issue to our Sevadaars to fix it before others face it.`,
+  GENERIC_ERROR_DESCRIPTION: `This is really unusual. Try clearing your cache. If you still face the issue, you may want to hit the Feedback button at the bottom of this page, and report this issue to our Sevadaars to fix it before others face it.`,
```

Comment:
Could we please make `clearing your cache` a link? Maybe this? https://geekermag.com/clear-browser-cache/